crates/re_components/src/coordinates.rs

```diff
@@ -105,6 +105,12 @@ impl re_log_types::Component for ViewCoordinates {
 }
 
 impl ViewCoordinates {
+    /// Default right-handed view coordinates of `re_renderer`: X=Right, Y=Up, Z=Back.
+    pub const RUB: Self = Self([ViewDir::Right, ViewDir::Up, ViewDir::Back]);
```

Comment:
Is there a hard constraint forcing this, or do we have the option to (in the future) make the default view coordinates of `re_renderer` RDF instead?  This seems like it would cut down on some of the book-keeping around whether a camera is a "real camera" vs a "re-renderer camera." Having one standard set of conventions for cameras would reduce ambiguity, and given that we position ourselves as a robotics-and-computer-vision product, I think RDF is the right choice here.

Reply:
I agree it would be nice to use RDF as the default everywhere. We could probably switch `re_renderer` with some work, though it should be noted that RDF is quite unusual for rendering. @Wumpf can probably add some more nuance here.